Formula/uptoc.rb

```diff
@@ -0,0 +1,25 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
```

Comment:
Please remove this line. Just noticed the documentation for GoReleaser's Homebrew formula generation [here](https://goreleaser.com/customization/homebrew/), and it's pretty neat, but the generated formulae generally don't meet our requirements (e.g. they ship the built binaries rather than facilitate building from source), hence the requested changes below.

Of course, the generated formula is still completely usable as-is [if you choose to host your own Homebrew tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).